app/views/notifier/team_hours_update.html.erb

```diff
@@ -17,7 +17,7 @@
       <tr>
         <td width="800px" height="180px" style="vertical-align: top;">
           <p>
-            For the week of <%= friendly_date_range(Date.this_week) %>, you have used <%= @billed_hours %> of <%= @team.hours %> budgeted hours.
+            For the week of <%= friendly_date_range(Date.this_week) %>, you have used <%= @billed_hours.round(2) %> of <%= @team.hours %> budgeted hours.
```

Comment:
`team.hours` is exactly what's entered in the text field on the edit page. It should never be a problem. Also that field is an integer and won't support decimal values.